dask/bytes/tests/test_http.py

```diff
@@ -113,8 +113,18 @@ def test_files(dir_server):
             assert f.read() == open(os.path.join(dir_server, f2), 'rb').read()
 
 
+def test_open_glob(dir_server):
+    root = 'http://localhost:8999/'
+    fs = open_files(root + '/*')
+    assert fs[0].path == 'http://localhost:8999/a'
+    assert fs[1].path == 'http://localhost:8999/b'
+
+
 @pytest.mark.network
 def test_parquet():
+    from distutils.version import LooseVersion
+    if LooseVersion(requests.__version__) < LooseVersion("2.21.0"):
+        pytest.skip()
```

Comment:
This is because older requests seems not to be able to make a HEAD request successfully to find out how big the file is.